patches/chrome-services-printing-printing_service.cc.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/chrome/services/printing/printing_service.cc b/chrome/services/printing/printing_service.cc
+index ac5f433d983fc20d9f24773ef7474f64a1fbfcb4..41d92bf742403fbdf5c96a8bbee6b770096ce382 100644
+--- a/chrome/services/printing/printing_service.cc
++++ b/chrome/services/printing/printing_service.cc
+@@ -15,7 +15,7 @@
+ #include "chrome/services/printing/pdf_flattener.h"
+ #endif
+ 
+-#if BUILDFLAG(IS_CHROMEOS_ASH) || BUILDFLAG(IS_WIN)
++#if BUILDFLAG(IS_CHROMEOS_ASH) || BUILDFLAG(IS_WIN) || BUILDFLAG(ENABLE_PRINT_PREVIEW)
```

Comment:
@goodov Should we do our own brave printing service to include pdf_to_bitmap converter instead of extending upstream one? We won't need these build flag patches if we do so or is there better way to reduce these kind of patches?

Reply:
> Should we do our own brave printing service to include pdf_to_bitmap converter instead of extending upstream one? We won't need these build flag patches if we do so or is there better way to reduce these kind of patches?

the current approach does not look right. You're enabling a bunch of stuff that's not needed on other platforms. It's unclear what security concerns this adds into Brave, so I'd rather avoid that.

a separate mojo utility here to just serialize pdfs to bitmap looks better.

Reply:
Actually they are not needed after series of changes so I removed them here
956338a87e6ac2375967d247e5028ed75dd864ae
It was added due to a DCHECK failure when running the service but now I can't reproduce it anymore with our latest codes.

Reply:
thanks for catching it, fixed and squashed.